drakrun/drakrun/apiscout.py

```diff
@@ -0,0 +1,116 @@
+import json
+import logging
+from operator import attrgetter
+from pathlib import Path, PureWindowsPath
+import pefile
+from typing import List, Dict, Any
+
+log = logging.getLogger(__name__)
+
+
+def get_bitness(pe: pefile.PE) -> int:
+    if pe.FILE_HEADER.Machine == pefile.MACHINE_TYPE["IMAGE_FILE_MACHINE_AMD64"]:
+        return 64
+    elif pe.FILE_HEADER.Machine == pefile.MACHINE_TYPE["IMAGE_FILE_MACHINE_I386"]:
+        return 32
+    else:
+        log.error(
+            f"Unsupported machine_type: {pe.FILE_HEADER.Machine} -> {pefile.MACHINE_TYPE[pe.FILE_HEADER.Machine]}"
+        )
+        return 0
```

Comment:
We should raise an exception here or return `None`. Value "0" means nothing in this context.